packages/build-utils/src/convert-runtime-to-plugin.ts

```diff
@@ -305,12 +309,14 @@ export function convertRuntimeToPlugin(
         ...newDirectoriesEntrypoint,
       ]);
 
-      const apiRouteHandler = `${parse(entry).name}.${handlerMethod}`;
-
       // Add an entry that will later on be added to the `functions-manifest.json`
       // file that is placed inside of the `.output` directory.
-      pages[entrypoint] = {
-        handler: apiRouteHandler,
+      pages[join(entryDir, entryBase)] = {
```

Comment:
`join()` here is going to use the windows backslash on windows and I think we want to use `/`

https://github.com/vercel/vercel/runs/4413823572?check_suite_focus=true

Reply:
I think you can use `path.posix.join()` instead

Reply:
Fixed!